R-package/README.md

```diff
@@ -5,15 +5,44 @@
 ### Contents
 
 * [Installation](#installation)
+    - [Installing the CRAN package](#installing-the-cran-package)
+    - [Installing from Source with CMake](#installing-from-source-with-cmake)
+    - [Installing a GPU-enabled Build](#installing-a-gpu-enabled-build)
+    - [Installing Precompiled Binaries](#installing-precompiled-binaries)
+    - [Installing from a Pre-compiled lib_lightgbm](#lib_lightgbm)
 * [Examples](#examples)
 * [Testing](#testing)
-* [Preparing a CRAN Package and Installing It](#preparing-a-cran-package-and-installing-it)
+* [Preparing a CRAN Package](#preparing-a-cran-package)
 * [External Repositories](#external-unofficial-repositories)
 * [Known Issues](#known-issues)
 
 Installation
 ------------
 
+For the easiest installation, go to ["Installing the CRAN package"](#installing-the-cran-package).
+
+If you experience any issues with that, try ["Installing from Source with CMake"](#install). This can produce a more efficient version of the library on Windows systems with Visual Studio.
+
+To build a GPU-enabled version of the package, follow the steps in ["Installing a GPU-enabled Build"](#installing-a-gpu-enabled-build)
+
+If any of the above options do not work for you or do not meet your needs, please let the maintainers know by [opening an issue](https://github.com/microsoft/LightGBM/issues).
+
+When your package installation is done, you can check quickly if your LightGBM R-package is working by running the following:
+
+```r
+library(lightgbm)
+data(agaricus.train, package='lightgbm')
+train <- agaricus.train
+dtrain <- lgb.Dataset(train$data, label = train$label)
+model <- lgb.cv(
+    params = list(
+        objective = "regression"
+        , metric = "l2"
+    )
+    , data = dtrain
+)
+```
+
 ### Installing the CRAN package
 
 As of this writing, `LightGBM`'s R package is not available on CRAN. However, start with `LightGBM` 3.0.0, you can install a released source distribution. This is the same type of package that you'd install from CRAN. It does not require `CMake`, Visual Studio, or anything else outside the CRAN toolchain.
```

Comment:
```suggestion
As of this writing, `LightGBM`'s R package is not available on CRAN. However, start with `LightGBM` 3.0.0, you can install a released source distribution. This is the same type of package that you'd install from CRAN. It does not require `CMake`, `Visual Studio`, or anything else outside the CRAN toolchain.
```

Reply:
Cannot comment on this line, but we need dot here:

![image](https://user-images.githubusercontent.com/25141164/92407397-2c815e00-f143-11ea-97b1-4b0169a83318.png)